z3c/dependencychecker/tests/conftest.py

```diff
@@ -86,31 +83,21 @@ def _add_egg_info(folder):
 
 
 def _write_pkg_info_file(folder):
-    with open(os.path.join(folder, "PKG-INFO"), "w") as pkg_info:
-        lines = "\n".join(
-            ["Metadata-Version: 1.0", "Name: testpackage", "Version: 1.0.dev0"]
-        )
-        pkg_info.write(lines)
+    lines = "\n".join(
+        ["Metadata-Version: 1.0", "Name: testpackage", "Version: 1.0.dev0"]
+    )
+    (folder / "PKG-INFO").write_text(lines)
```

Comment:
Not something that needs changing here, just a generic observation on my own habits.
My initial thought when I saw `.write_text(lines)` was "hey, can I put a list of strings into `.write_text()`? Reason is that I *nowadays* normally use "text" or "contents" for a multi-line string. `lines`, for me, is always a list of individual lines (=strings).

Now I'm wondering if I originally put in that `lines` years and years ago and whether I changed my habits :-)

Reply:
I think, I rewrote the tests to use pytest long ago, so it might be my habits here 😄